.github/workflows/test_dev.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           use-mamba: true
 
       - name: Generate env spec
-        run: pip install tomli && python make_env.py test --py-version ${{ matrix.python-version }}
+        run: pip install tomli && python make_env.py test --py-version ${{ matrix.python-version }}.*
 
```

Comment:
I have a slight susspiscion this is going to break since `.*` is a glob in bash, so it is probably going to expand to things you don't expect. I think the `make_env.py` in core now has a update to deal with this, if not please ping me and I'll make one. If you want to keep this patern you should wrap it in quotes at least